### PR TITLE
Improved Great People IV

### PR DESCRIPTION
**Pull request by etiennefd**
Highlights: 
- Maya names are now separate from Aztecs. They include a few Toltec names and a few mythological figures to fill the gaps. 
- Modern Greek names, so that Plato and Homer aren't born after the Greeks respawn in the 19th c. 
- Many new Scandinavian names, including those in Merijn's Sweden mod

(also my text editor removes trailing whitespace, which is annoying and leads to tons of meaningless changes in the git comparison, but I have disabled that feature for next time)

### CODE REVIEW

File: Assets/Python/GreatPeople.py [hunk 1]
```diff
@@ -444,27 +454,49 @@ def setup():
...
+		"Iannis Xenakis", # 20th
+		"Mikis Theodorakis", # 20th
 	],
 	iGreatScientist : [
 		"Pythagoras", # 6th BC
 		"Sokrates", # 5th BC
 		"Demokritos", # 5th BC
 		"Anaxagoras", # 5th BC
 		"Hippokrates", # 5th BC
-		iClassical,
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py:454]: Why did you remove this?

File: Assets/Python/GreatPeople.py [hunk 2]
```diff
@@ -516,7 +568,7 @@ def setup():
 		iClassical,
 		"Mago", # 4th BC
 		"Hasdrubal Clitomachus", # 2nd BC
-		"Abba", # 3rd 
+		"Abba", # 3rd
```
Comment (dguenms) [inline Assets/Python/GreatPeople.py:571]: Now a Swedish great person? :D

**Review by dguenms — CHANGES_REQUESTED** [review]
Great contributions as always, only one small question.

**Comment by dguenms** [comment]
Oh by the way, I don't mind removing trailing white spaces.

File: Assets/Python/GreatPeople.py (hunk 1 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py:454]: I think all the names above the iClassical line are Classical Era people. In all other Greek GP categories, people from the 6th and 5th century BC are considered classical era.

File: Assets/Python/GreatPeople.py (hunk 2 above)
Comment (etiennefd) [reply Assets/Python/GreatPeople.py:571]: Haha, I have no idea who that is

Comment (dguenms) [reply Assets/Python/GreatPeople.py:571]: Me neither, I think the name comes from the original civ specific GP name mod.

File: Assets/Python/GreatPeople.py (hunk 1 above)
Comment (dguenms) [reply Assets/Python/GreatPeople.py:454]: Oh okay, yeah that makes sense.

**Review by dguenms — APPROVED** [review]
